docs/dietpi_tools.md

```diff
@@ -2,142 +2,170 @@
 
 ## DietPi-Launcher
 
+**dietpi-software** will be automatically ran on the first login. It could be accessed at any time running next next from the command line:
```

Comment:
I think `dietpi-launcher` and `dietpi-software` git mixed up here? The latter is called on first login (so below is correct) but `dietpi-launcher` is not.

Reply:
Correct. Thanks, I will fix it